frontend/src/LoginForm.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { Formik, Form } from "formik";
+import * as yup from "yup";
+
+import EmailInput from "./EmailInput";
+import PasswordInput from "./PasswordInput";
+import Button from "./Button";
+//const axios = require('axios')
+var axios = require("axios");
+var MockAdapter = require("axios-mock-adapter");
+var mock = new MockAdapter(axios);
+
+
+
+
+const schema = yup.object().shape({
+    email: yup
+        .string()
+        .required("E-mail megadása kötelező!")
+        .email("Nem megfelelő email cím!"),
+    password: yup
+        .string()
+        .required("A jelszó megadása kötelező!")
+        .min(5, "A jelszó legalább 5 karakter legyen!")
+});
+
+mock.onGet("http://localhost:8080/api/users/login", { username: 'user12345', password: 'user12345' }).reply(204, { accessToken: '...' });
```

Comment:
A mockolást is egy külön file-ba raknám, amit a userAPI.js húzhatna be magának vagy pedig a userAPI.js-en belül lehetne közvetlenül.

Reply:
Tomi: "Min. 8 karakter, tartalmazzon kis- es nagybetut, szamot " 
Átállíottam 8-ra. Az issue-ban azt kérted, a validációnál arra kell figyelni, hogy legyen elég hosszú a jelszó. 
Mivel ez login, itt a többi kritériumra nem kell figyelni?

Reply:
Kiraktam UserAPI.js-be.